.github/workflows/lint.yml

```diff
@@ -136,3 +136,22 @@ jobs:
         uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # v2
       - name: Running check_path_lengths
         run: ./build_tools/scripts/check_path_lengths.py
+
+  generated_cmake_files:
```

Comment:
Can you add this also to https://github.com/iree-org/iree/blob/main/build_tools/scripts/lint.sh

Reply:
nice auto-merge :joy: